config/settings.py

```diff
@@ -10,7 +10,9 @@
 https://docs.djangoproject.com/en/5.2/ref/settings/
 """
 
+import os
 from pathlib import Path
+from typing import List
```

Comment:
As of Python 3.9, type aliases like List, Tuple, Dict, ... are [deprecated](https://docs.python.org/3/library/typing.html#deprecated-aliases).
So, from then on, use the built-in types list, tuple, dict, ...